packages/theme/src/action-menu/adaptation.less

```
@@ -0,0 +1,21 @@
+@import './vars.less';
+
+@action-menu-prefix-cls: ~'@{css-prefix}action-menu';
+@dropdown-prefix-cls: ~'@{css-prefix}dropdown';
+@dropdown-item-prefix-cls: ~'@{css-prefix}dropdown-item';
+
+@media (max-width: 1280px) {
+  .tiny-dropdown-menu {
+    min-width: 0;
+  }
+
+  .@{dropdown-item-prefix-cls} {
```

Stylelint (17.3.0)

[error] Unknown word dropdown-item-prefix-cls (CssSyntaxError)

(CssSyntaxError)